packages/react/src/components/actions/LinkAction.tsx

```diff
@@ -23,6 +23,15 @@ export class LinkAction extends Component<ILinkActionProps> {
             opts.rel = 'noopener noreferrer';
         }
 
-        return <a className={actionClasses} href={href} title={this.props.action.name} role="button" {...opts} />;
+        return (
+            <a
+                className={actionClasses}
+                href={href}
+                title={this.props.action.name}
+                role="button"
+                {...opts}
+                onClickCapture={this.props.action.onClickCapture}
```

Comment:
I'm wondering if we should add the onClickCapture to other types of actions. E.g., a dev using the `trigger` or `subActions` could see in the autocomplete that onClickCapture is available and try to use it but it won't work.

Since it's deprecated we could also simply add a JSDoc comment to say it only works with `link`. What do you think?

Reply:
Good point. I was also under the impression that it would be good enough to only support my use-case since the component is deprecated. I added a JSDoc on the option.